lib/facter/network.rb

```diff
@@ -12,8 +12,8 @@ def facter_3
 
 # Gateway
 # Expected output: The ip address of the nexthop/default router
-Facter.add('network_nexthop_ip') do
-  confine :kernel => :linux # rubocop:disable Style/HashSyntax
+Facter.add(:network_nexthop_ip) do
+  confine kernel: :linux
```

Comment:
Can you confine on symbols? I'd have thought it would need to match on strings. Maybe it doesn't care, but I've always been unclear about that.